packages/select/src/components/select/_select.scss

```diff
@@ -29,3 +29,12 @@ $select-popover-max-width: $pt-grid-size * 40 !default;
     }
   }
 }
+
+.full-width {
```

Comment:
I think this could be nested inside the `.#{$ns}-select-popover { }` block for more clarity -- it's a kind of modifier for the popover.

```scss
.#{$ns}-select-popover {
  // ...

  &.#{$ns}-select-match-target-width {
    // ...
  }
}
```